src/css/components/Button.scss

```diff
@@ -166,9 +161,17 @@ $Button--small-padding: 0 $su-small;
   padding: $Button--small-padding;
 }
 
-/**
- * Modifier: Button Large
- */
+// * Modifier: Button Square small
```

Comment:
nice, I like this for the icon buttons

Reply:
If in the future we want default and large icon buttons, we can add those too.